packages/flutter/lib/src/gestures/scale.dart

```diff
@@ -2,6 +2,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'dart:math' as math;
 import 'arena.dart';
```

Comment:
really trivial nit: there should be a blank line between the `dart:` imports and the relative imports.